### PR TITLE
Clean up local development environment

### PR DESCRIPTION
**Pull request by maddybvh**
- Deletes all files that should be ignored by git.
- Use only yarn for version dependency
- Updates the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,13 @@
...
-Looking for more guidance? Full documentation for Gatsby lives [on the website](https://www.gatsbyjs.org/). Here are some places to start:
-
-- **For most developers, we recommend starting with our [in-depth tutorial for creating a site with Gatsby](https://www.gatsbyjs.org/tutorial/).** It starts with zero assumptions about your level of ability and walks through every step of the process.
-
-- **To dive straight into code samples, head [to our documentation](https://www.gatsbyjs.org/docs/).** In particular, check out the _Guides_, _API Reference_, and _Advanced Tutorials_ sections in the sidebar.
-
-## 💫 Deploy
-
-[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/gatsbyjs/gatsby-starter-default)
+3. run `gatsby develop` and navigate to http://localhost:8000/
```
Comment (madinthebull) [inline README.md:9]: `yarn develop` instead!

**Review by madinthebull — APPROVED** [review]
Looks good! One change to the Readme, but it doesn't need to be in this PR.

File: README.md (hunk 1 above)
Comment (maddybvh) [reply README.md:9]: Are you sure? I was running `gatsby develop`

Comment (madinthebull) [reply README.md]: I went ahead and updated it!

**Pull request merged**